services/matching-service/domain/match_validators.ts

```diff
@@ -0,0 +1,42 @@
+import { queues } from "../start";
+
+
+export function extractDifficulty(difficulty: string) {
+    const DIFFICULTIES = [
+        "easy",
+        "medium",
+        "hard"
+    ]
```

Comment:
Would be good to extract these enums out into a separate file called `types.ts`, so that they can be reused

Reply:
Done